### PR TITLE
Fix typo for pending channel in 9.2.7.

### PR DESCRIPTION
**Pull request by jwhui**
Fixes a bug reported by @nibanks.

### CODE REVIEW

**Comment by bukepo** [comment]
LGTM👍

**Comment by xiaom-GitHub** [comment]
LGTM 👍

File: tests/scripts/thread-cert/Cert_9_2_07_DelayTimer.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 ROUTER_PENDING_TIMESTAMP = 30
 ROUTER_PENDING_ACTIVE_TIMESTAMP = 21
```
Comment (jwhui) [inline tests/scripts/thread-cert/Cert_9_2_07_DelayTimer.py]: When the Pending Dataset Delay Timer expires, its contents are applied to the Active Dataset, including the Active Timestamp itself.

Comment (xiaom-GitHub) [inline tests/scripts/thread-cert/Cert_9_2_07_DelayTimer.py]: I see. Thanks.

**Pull request merged**